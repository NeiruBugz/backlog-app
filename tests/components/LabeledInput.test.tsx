import { LabeledInput } from '@components';
import { render } from '@testing-library/react';

const props = {
  label: 'Snapshot',
  inputId: 'snapshot',
  inputType: 'text',
  inputName: 'snapshot',
};

describe('LabeledInput', () => {
  it('snapshot', () => {
    const { container } = render(<LabeledInput {...props} />);
    expect(container).toMatchSnapshot();
  });

  it('snapshot with classes', () => {
    const { container } = render(<LabeledInput {...props} />);
    expect(container).toMatchSnapshot();
  });
});