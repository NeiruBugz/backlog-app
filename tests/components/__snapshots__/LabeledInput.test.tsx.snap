// Vitest Snapshot v1

exports[`LabeledInput > snapshot 1`] = `
<div>
  <label
    class="ba-label"
    for="snapshot"
  >
    Snapshot
    <input
      class="ba-input"
      id="snapshot"
      name="snapshot"
      type="text"
    />
  </label>
</div>
`;

exports[`LabeledInput > snapshot with classes 1`] = `
<div>
  <label
    class="ba-label"
    for="snapshot"
  >
    Snapshot
    <input
      class="ba-input"
      id="snapshot"
      name="snapshot"
      type="text"
    />
  </label>
</div>
`;
